web/pdf_document_properties.js

```diff
@@ -87,71 +113,71 @@ class PDFDocumentProperties {
    * @param {Object} pdfDocument - A reference to the PDF document.
    * @param {string} url - The URL of the document.
    */
-  setDocumentAndUrl(pdfDocument, url) {
+  setDocument(pdfDocument, url) {
+    if (this.pdfDocument) {
```

Comment:
The checks are fine then, but a small comment could indeed be helpful here to avoid any confusion.

I'm fine with reverting the naming changes. It will make the diff smaller and for now it does not add much. We can change all of them at a later time if we want, but then we should indeed do all of them instead of just a few.